ThirdParty/paraview/CMakeLists.txt

```diff
@@ -0,0 +1,10 @@
+file(GLOB_RECURSE SOURCES "${CMAKE_CURRENT_SOURCE_DIR}/*.cxx")
+
+set(TARGET ThirdPartyParaView)
+add_library(${TARGET} OBJECT ${SOURCES})
+if (VTK_VERSION VERSION_LESS "8.90.0")
+    target_include_directories(${TARGET} SYSTEM ${VTK_INCLUDE_DIRS})
```

Comment:
This line gives me cmake error when I use VTK8.2. 
```
CMake Error at ThirdParty/paraview/CMakeLists.txt:6 (target_include_directories):
  target_include_directories called with invalid arguments
```
If I change it to just
```cmake
include_directories(SYSTEM ${VTK_INCLUDE_DIRS})
```
it seems to work just fine.

Reply:
Please try again with the revision.